prow/plugins/config_test.go

```diff
@@ -216,16 +216,22 @@ func TestSetApproveDefaults(t *testing.T) {
 
 		commandHelpLink         string
 		expectedCommandHelpLink string
+		prProcessLink           string
+		expectedPrProcessLink   string
 	}{
 		{
 			name:                    "set commandHelpLink default",
```

Comment:
Please rename the two testcases to clarify that they test both commandHelpLink and PrProcessLink (or create new testcases for the latter)

Reply:
Renamed, thanks!